src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,14 @@
+body, h1, h2, h3, h4, h5, h6, ul, p {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h3, a {
+  transition: color $transition-duration;
+  -webkit-transition: color $transition-duration;
+}
```

Comment:
also for `a` you can reset
```
a {
  text-decoration: none;
}
```

so now you can delete this rule on your links in your files